packages/consent-proof-signature/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import Long from "long";
+import { invitation } from "@xmtp/proto";
+/**
+ *
+ * @param peerAddress - Ethereum address of the broadcaster
+ * @param timestampMs - Timestamp in milliseconds used in the signature
+ * @returns
+ */
+export const createConsentMessage = (
+  peerAddress: string,
+  timestampMs: number,
+): string =>
+  "XMTP : Grant inbox consent to sender\n" +
+  "\n" +
+  `Current Time: ${timestampMs}\n` +
```

Comment:
We should think about pretty printing the date. That way it's more obvious if it's set to anything other than now. Would have to be UTC, but still more legible than a bunch of digits.

Reply:
Can do, but will need to bump some of the other SDKs to support that